RockWeb/Themes/NewSpring/assets/Lava/UI/locations-information.lava

```diff
@@ -1,11 +1,9 @@
-{% assign campusSlug = 'Global' | PageParameter:'CampusSlug' | Replace:'-',' ' | Uppercase %}
-{% campus where:'Name == "{{ campusSlug }}"' iterator:'campuses' %}
-{% for campus in campuses %}
-
-{% assign location = campus.Location %}
-{% assign mailingAddress = campus | Attribute:'MailingAddress','Object' %}
-{% assign officeAddress = campus | Attribute:'OfficeAddress','Object' %}
-{% assign getDirections = campus | Attribute:'GetDirectionsURL','RawValue' %}
+{% assign campusSlug = 'Global' | PageParameter:'CampusSlug' | Replace:'-',' ' | Capitalize %}
+{% assign campuses = 'campuses' | PersistedDataset %}
+{% assign campus = campuses | Where:'Name', campusSlug | First %}
```

Comment:
Same here, we can combine 2 + 3 by just continuing to add the filters.
`{% assign campus = 'campuses' | PersistedDataset | Where:'Name', campusSlug | First %}`